pkg/metrics/producers/reservedcapacity/producer.go

```diff
@@ -21,7 +21,7 @@ import (
 	"strconv"
 
 	"github.com/awslabs/karpenter/pkg/apis/autoscaling/v1alpha1"
-	"github.com/awslabs/karpenter/pkg/utils/node"
+	"github.com/awslabs/karpenter/pkg/utils/nodeutils"
```

Comment:
This stutters. Can you think of a better solution?

Reply:
I can't. Scheduling is the only other possibility here, for me.

Reply:
Keep thinking -- I can help you brainstorm if the only option is stuttering. 

Off the top of my head, this doesn't stutter
```
utilsnode      github.com/awslabs/karpenter/pkg/utils/node
```
It also means that consumers don't need the extra prefix if they don't have local variables named node. 

Scheduling also seems viable -- this is deeply related to scheduling.

Reply:
I was thinking `instances` but that's not very clear.

Reply:
Instances is a very AWS specific word. e.g. GCP uses "machine". Kubernetes typically uses the term "node"

Reply:
Going with `utilsnode      github.com/awslabs/karpenter/pkg/utils/node`